src/meta/service/tests/it/grpc/metasrv_grpc_api.rs

```diff
@@ -157,10 +158,10 @@ async fn test_retry_join() -> anyhow::Result<()> {
 
         tc1.config.raft_config.join = vec![bad_addr.clone()];
         let ret = start_metasrv_with_context(&mut tc1).await;
-        let expect = MetaError::MetaServiceError(format!(
-            "join cluster via {:?} fail",
-            tc1.config.raft_config.join
-        ));
+        let expect = MetaManagementError::Join(AnyError::error(format!(
+            "fail to join {} cluster via {:?}",
+            1, tc1.config.raft_config.join
```

Comment:
Is this a typo, a normal `1`?

Reply:
In this test it tries to join node-1 to the cluster thus here it is just a `1`.